manager/state/raft/raft.go

```diff
@@ -914,7 +936,7 @@ func (n *Node) configure(ctx context.Context, cc raftpb.ConfChange) error {
 	case <-ctx.Done():
 		n.wait.trigger(cc.ID, nil)
 		return ctx.Err()
-	case <-n.stopCh:
+	case <-n.Ctx.Done():
```

Comment:
It's probably officially not an anti-pattern :/
https://github.com/golang/go/commit/4cad610401edc11fe921205438a7b3ab4faa3982